include/SANDTrackerUtils.h

```diff
@@ -0,0 +1,247 @@
+#ifndef SANDTrackerUTILS_H
+#define SANDTrackerUTILS_H
+
+#include <vector>
+#include <math.h>
+
+#include "TTreeReader.h"
+#include "TMatrixD.h"
+#include <TDecompChol.h>
+#include <TRandom3.h>
+
+#include "utils.h"
+
+#include "SANDTrackerDigitCollection.h"
+#include "SANDKFTrack.h"
+
+using SANDTrackANDKFStateCovarianceMatrix = TMatrixD;
+using SANDTrackANDKFMeasurement = TMatrixD;
+
+#define IS_VERBOSE false
+
+#if (IS_VERBOSE)
+#define SANDTrackerTRACKRECO_LOG(level, message)                                                                                                       \
+{                                                                                                                                              \
+  std::cout << std::setw(40) << std::left << __FUNCTION__ << " in " <<                                     \
+               std::setw(100) << std::left << TString::Format("%s:%d",__FILE__,__LINE__).Data() << " --> " \
+                              << level << ": " << message << std::endl;                                    \
+}
+#else
+#define SANDTrackerTRACKRECO_LOG(level, message)
+#endif
+
+class SANDTrackerUtils
+{
+ private:
+  static TGeoManager* fGeo;
```

Comment:
variable inside a class shuold follow a (the same) naming convention

please see: https://google.github.io/styleguide/cppguide.html#Variable_Names